docs/photutils/psf.rst

```diff
@@ -28,10 +28,379 @@ model-fitting analysis, regardless to exactly what kind of model is actually
+    sources = Table()
+    sources['flux'] = [700, 800, 700, 800]
+    sources['x_mean'] = [12, 17, 12, 17]
+    sources['y_mean'] = [15, 15, 20, 20]
+    sources['x_stddev'] = sigma_psf*np.ones(4)
+    sources['y_stddev'] = sources['x_stddev']
+    sources['theta'] = [0, 0, 0, 0]
+    sources['id'] = [1, 2, 3, 4]
+    tshape = (32, 32)
+    image = (make_gaussian_sources(tshape, sources) +
+             make_noise_image(tshape, type='poisson', mean=6.,
+                              random_state=1) +
+             make_noise_image(tshape, type='gaussian', mean=0.,
+                              stddev=2., random_state=1))
+
+    from matplotlib import rcParams
+    import matplotlib.pyplot as plt
+    rcParams['image.cmap'] = 'viridis'
+    rcParams['image.aspect'] = 1  # to get images with square pixels
+    rcParams['image.interpolation'] = 'nearest'
+    rcParams['image.origin'] = 'lower'
+    rcParams['font.size'] = 16
+    
+    plt.imshow(image)
+    plt.title('Simulated data')
+    plt.colorbar(orientation='horizontal', fraction=0.046, pad=0.04)
+
+
+Then let's import the required parts to set up a `~photutils.psf.DAOPhotPSFPhotometry` object::
+
+    >>> from photutils.detection import IRAFStarFinder
+    >>> from photutils.psf import IntegratedGaussianPRF, DAOGroup
+    >>> from photutils.background import MMMBackground, MADStdBackgroundRMS
+    >>> from astropy.modeling.fitting import LevMarLSQFitter
+    >>> from astropy.stats import gaussian_sigma_to_fwhm
+
+Let's then instantiate the objects::
+
+    >>> bkgrms = MADStdBackgroundRMS()
+    >>> std = bkgrms(image) # doctest: +SKIP
+    >>> iraffind = IRAFStarFinder(threshold=3.5*std,
+    ...                           fwhm=sigma_psf*gaussian_sigma_to_fwhm,
+    ...                           minsep_fwhm=0.01, roundhi=5.0, roundlo=-5.0,
+    ...                           sharplo=0.0, sharphi=2.0) # doctest: +SKIP
+    >>> daogroup = DAOGroup(2.0*sigma_psf*gaussian_sigma_to_fwhm)
+    >>> mmm_bkg = MMMBackground()
+    >>> fitter = LevMarLSQFitter()
+    >>> psf_model = IntegratedGaussianPRF(sigma=sigma_psf) # doctest: +SKIP
+
+Note that the parameters valeus for the finder class, i.e.,
+`~photutils.detection.IRAFStarFinder`, are completly chosen in an arbitrary
+manner and optimum values do vary according to the data. 
+
+Now, we can create a `~photutils.psf.DAOPhotPSFPhotometry` object::
+
+    >>> from photutils.psf import DAOPhotPSFPhotometry
+    >>> daophot_photometry = DAOPhotPSFPhotometry(finder=iraffind,
+    ...                                           group_maker=daogroup,
+    ...                                           bkg_estimator=mmm_bkg,
+    ...                                           psf_model=psf_model,
+    ...                                           fitter=LevMarLSQFitter(),
+    ...                                           niters=1, fitshape=(11,11)) # doctest: +SKIP
+
+As mention before, one can use the ``daophot_photometry`` object as a function
```

Comment:
"one can use" -> "the way to actually *do* the photometry" or something like that.  That is, get across the idea that it's not an option, but rather the actually most important part of the whole process - it's where the photometric measurements actulally happen.